front/src/modules/timetableItem/components/ManageTimetableItem/helpers/checkCurrentConfig.ts

```diff
@@ -112,6 +112,17 @@ const checkCurrentConfig = (
         })
       );
     }
+    if (timeWindow.total('minute') >= 43800) {
```

Comment:
Now you can use `MAX_TIMEWINDOW_MINUTES`here ;)
```suggestion
    if (timeWindow.total('minute') >= MAX_TIMEWINDOW_MINUTES) {
```